app/routers/subscription.py

```diff
@@ -63,6 +63,10 @@ def user_subscription(
                 {"user": user}
             )
         )
+    
+    if len(user.sub_tags) > 0 and len(user.sub_url_prefix) > 0:
+        from app.routers.clash import generate_subscription
```

Comment:
also your importing from routers in here, if you just follow project structure and put your code in right place there is no need to make unnecessary dependencies

Reply:
I want to compat with current subscription of Marzban